src/components/hero-section.tsx

```
@@ -0,0 +1,117 @@
+          >
+            Ismail Chabane
+          </h1>
+        </div>
+        <div
+          ref={descriptionRef}
+          data-cursor="description"
+          className=" opacity-0 space-y-2"
+        >
+          <p className="text-2xl font-medium">Software Engineer & Freelancer</p>
+          <p className="text-xl text-gray-600">
+            pursuing Master's in Data Science
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)